content/shmem_team_broadcast.tex

```diff
@@ -0,0 +1,88 @@
+\apisummary{
+    Broadcasts a block of data from one \ac{PE} to one or more destination
+    \acp{PE}.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_team\_broadcast32}@(shmem_team_t team, void *dest, const void *source, size_t nelems, int PE_root);
+void @\FuncDecl{shmem\_team\_broadcast64}@(shmem_team_t team, void *dest, const void *source, size_t nelems, int PE_root);
+\end{Csynopsis}
+
+\begin{apiarguments}
+
+\apiargument{IN}{team}{A valid SHMEM team handle to a team which has been created with support for collective operations.}
+\apiargument{OUT}{dest}{A symmetric data object.} 
+\apiargument{IN}{source}{A symmetric data object that can be of any data type
+    that is permissible for the \dest{} argument.}
+\apiargument{IN}{nelems}{The number of elements in \source.  For
+    \FUNC{shmem\_team\_broadcast32}, this is the number of
+    32-bit halfwords.  nelems must be of type \VAR{size\_t} in \Cstd.}
+\apiargument{IN}{PE\_root}{Zero-based ordinal of the \ac{PE}, with respect to
+    the team, from which the data is copied. Must be greater than or equal to
+    0 and less than the result of calling \FUNC{shmem\_team\_n\_pes(team)}.
+    \VAR{PE\_root} must be of type integer.} 
+
+\end{apiarguments}
+
+\apidescription{   
+    \openshmem broadcast routines are collective routines over an existing team.
+    They copy data object \source{} on the processor specified by \VAR{PE\_root}
+    and store the values at \dest{} on the other \acp{PE} that are members of the
+    team. The data is not copied to the \dest{} area on the root \ac{PE}.
+    
+    As with all \openshmem team collective routines, each of these routines assumes that
+    only \acp{PE} in the given team call the routine.  If a \ac{PE} not in the
+    team calls an \openshmem team collective routine, the behavior is undefined.
+
```

Comment:
Doesn't this statement seem incorrect - I suppose the behavior can be defined? A PE not part of the team should either have SHMEM_TEAM_NULL and or NULL as the team handle. We could define as these are not permitted.

Reply:
Good point. This is a hangover from the description for broadcast on a set. This should be removed.